cpp/src/plasma/CMakeLists.txt

```diff
@@ -19,16 +19,13 @@ cmake_minimum_required(VERSION 2.8)
 
 project(plasma)
 
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/../python/cmake_modules")
```

Comment:
We could also move these `cmake_modules` to the top-level so that they are available to any `arrow-cpp` component.